recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "Sophus" %}
-{% set version = "22.10" %}
+{% set version = "1.24.6" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
```

Comment:
I wonder if we should use an epoch since the versioning scheme was changed / fixed?

E.g. `1!1.24.6` should be `>22.10` while otherwise `22.10` would win against `1.24.6`.

Reply:
Just FYI tested with `conda`'s VersionOrder:

<img width="370" alt="Screenshot 2024-06-20 at 11 24 56" src="https://github.com/conda-forge/sophus-feedstock/assets/885054/e56cb123-1850-40f6-83f9-4f54dc530028">

Reply:
TIL, epoch in versioning. Thank you for the suggestion and the testing.